lib/src/component/abstract_transition.dart

```diff
@@ -271,7 +267,7 @@ abstract class AbstractTransitionComponent<T extends AbstractTransitionProps,
 
   @mustCallSuper
   @override
-  void componentDidUpdate(Map prevProps, Map prevState) {
+  void componentDidUpdate(Map prevProps, Map prevState, [dynamic snapshot]) {
```

Comment:
#supernit I know thing were just moved around, but I noticed this is an unused parameter 🤷‍♂ 
```suggestion
  void componentDidUpdate(Map prevProps, Map prevState, [_]) {
```